doc/man3/SSL_CTX_set_split_send_fragment.pod

```diff
@@ -99,15 +105,59 @@ greater than the default that would have been used anyway. The normal default
 value depends on a number of factors but it will be at least
 SSL3_RT_MAX_PLAIN_LENGTH + SSL3_RT_MAX_ENCRYPTED_OVERHEAD (16704) bytes.
 
+SSL_CTX_set_tlsext_max_fragment_length() sets the default maximum fragment
+length negotiation mode via value B<mode> to B<ctx>.
+This setting affects only SSL instances created after this function is called.
+
+SSL_set_tlsext_max_fragment_length() sets the maximum fragment length
+negotiation mode via value B<mode> to B<ssl>.
+This setting will be used during a Handshake when extensions are exchanged
+between client and server.
+So it only affects SSL sessions created after this function is called.
+It affect the client-side as only its side may initiate this extension use.
```

Comment:
Doesn't this last comment also apply to SSL_CTX_set_tlsext_max_fragment_length() above? (Also - typo: affects)